process/process.go

```diff
@@ -403,9 +550,9 @@ func (p *process) pathAccess(path string, mode domain.AccessMode) error {
 	// Determine the start point.
 	var start string
 	if filepath.IsAbs(path) {
-		start = p.root
+		start = p.hroot
```

Comment:
Question: why p.hroot and not p.root?

Reply:
See that i'm not changing the behavior here. Before this PR, "p.root" held the path as seen from the host (/proc/<pid>/root). Now that same value is carried by the "p.hroot" parameter. I'm now utilizing "p.root" for the process' root attribute.